mobile/src/modules/i18n/translations/ja.json

```diff
@@ -124,102 +124,102 @@
     "missing": "なし",
     "from": "From",
```

Comment:
This is untranslated; unsure if there's no good translation for `From` in Japanese?

Reply:
I don't know where `From` is used.

Reply:
It's used on the "Unmatched Route" screen; we display the route we couldn't find and show the route we were previously on in the "From".